src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java

```diff
@@ -780,7 +780,7 @@ public void setGetCertifiedDismissed() {
 		//set certification message visible=false for a year
 		Date yearFromNow = new Date();
 		CalendarUtil.addMonthsToDate(yearFromNow, 12);
-		cookies.setCookie(USER_PROFILE_CERTIFICATION_VISIBLE_STATE_KEY, Boolean.toString(false), yearFromNow);
+		cookies.setCookie(USER_PROFILE_CERTIFICATION_VISIBLE_STATE_KEY + "." + currentUserId, Boolean.toString(false), yearFromNow);

```

Comment:
I repaired the tests.

On Fri, Apr 17, 2015 at 1:24 PM, Jay Hodgson <notifications@github.com>
wrote:

> In
> src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java
> <https://github.com/Sage-Bionetworks/SynapseWebClient/pull/1599#discussion_r28627197>
> :
>
> > @@ -780,7 +780,7 @@ public void setGetCertifiedDismissed() {
> >  		//set certification message visible=false for a year
> >  		Date yearFromNow = new Date();
> >  		CalendarUtil.addMonthsToDate(yearFromNow, 12);
> > -		cookies.setCookie(USER_PROFILE_CERTIFICATION_VISIBLE_STATE_KEY, Boolean.toString(false), yearFromNow);
> > +		cookies.setCookie(USER_PROFILE_CERTIFICATION_VISIBLE_STATE_KEY + "." + currentUserId, Boolean.toString(false), yearFromNow);
>
> update ProfilePresenterTest
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sage-Bionetworks/SynapseWebClient/pull/1599/files#r28627197>
> .
>